packages/pieces/sendy/src/lib/actions/unsubscribe.ts

```diff
@@ -0,0 +1,34 @@
+import { createAction, Property, Validators } from "@activepieces/pieces-framework";
+import { unsubscribe } from "../api";
+import { buildListDropdown } from "../props";
+import { sendyAuth, SendyAuthType } from "../auth";
+
+export const unsubscribeAction = createAction({
+	name        : 'unsubscribe',
+	auth        : sendyAuth,
+	displayName : 'Unsubscribe',
```

Comment:
Can you please rename it to `Remove Subscriber`

Reply:
No. This does not remove a subscriber. That would be delete. I think that this needs to be named unsubscribe. There is a difference.